tests/test_lamp_motion.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+# modules under test
+from pcdsdevices.lamp_motion import LAMP, LAMPMagneticBottle
+
+MOTOR1 = 'TMO:LAMP:MMS:01'
+MOTOR2 = 'TMO:LAMP:MMS:02'
+MOTOR3 = 'TMO:LAMP:MMS:03'
+MOTOR4 = 'TMO:LAMP:MMS:04'
+MOTOR5 = 'TMO:LAMP:MMS:05'
+MOTOR6 = 'TMO:LAMP:MMS:06'
+MOTOR7 = 'TMO:LAMP:MMS:07'
+MOTOR8 = 'TMO:LAMP:MMS:08'
+MOTOR9 = 'TMO:LAMP:MMS:09'
+MOTOR10 = 'TMO:LAMP:MMS:10'
+
+def test_lamp():
+    """Make sure we have assigned correct motors pvs we expect"""
+    lamp = LAMP(prefix='TMO:LAMP', name='test')
+    assert getattr(lamp, 'gas_jet_x').prefix == MOTOR1
```

Comment:
Nitpick
```suggestion
    assert lamp.gas_jet_x.prefix == MOTOR1
```

Reply:
It's probably best to just remove the `test_lamp_motion` file. When a test suite is just "is my code exactly what I wrote in my code" it becomes a maintenance hazard more than a self-check.

Reply:
Also note that this is instantiating an object with real PV connections to real devices inside of a test suite! This could potentially be hazardous if something is written wrong and this is run on the tmo subnet.

Reply:
For reference - see others that use `make_fake_device()`

Reply:
True, mocks would be better here.  It was mostly a self check.  I'll remove the test file, the lamp_motion devices don't really fit into the framework I see here.